boards/apollo3/lora_things_plus/src/main.rs

```diff
@@ -155,10 +165,80 @@ struct LoRaThingsPlus {
     temperature: &'static TemperatureDriver,
     humidity: &'static HumidityDriver,
     air_quality: &'static capsules_extra::air_quality::AirQualitySensor<'static>,
+    rng: Option<
+        &'static capsules_core::rng::RngDriver<
+            'static,
+            capsules_core::rng::Entropy32ToRandom<
+                'static,
+                capsules_extra::atecc508a::Atecc508a<'static>,
+            >,
+        >,
+    >,
     scheduler: &'static RoundRobinSched<'static>,
     systick: cortexm4::systick::SysTick,
 }
 
+#[cfg(feature = "atecc508a")]
+fn atecc508a_wakeup() {
+    let peripherals = unsafe { PERIPHERALS.unwrap() };
```

Comment:
```suggestion
    let peripherals = (unsafe { PERIPHERALS }).unwrap();
```

To more clearly denote that accessing the static is the unsafe part here.